deployment/create_docker_image.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+INDEX_REPO="clangd/llvm-remote-index"
+OUTPUT_NAME="clangd-linux-snapshot.zip"
+IMAGE_NAME="llvm-remote-index-server"
+INDEX_ASSET_PREFIX="llvm-index"
+INDEX_FILE_NAME="llvm.idx"
+SERVER_ASSET_PREFIX="clangd-linux-snapshot"
+INDEXER_PROJECT_ROOT="/home/runner/work/indexing-tools/indexing-tools/llvm-project/"
+
+TEMP_DIR="$(mktemp -d)"
+# Make sure we delete TEMP_DIR on exit.
+trap "rm -r $TEMP_DIR" EXIT
+
+# Abort script on failure.
+set -e
+# Print commands as we execute them.
+set -x
+
+cp deployment/Dockerfile "$TEMP_DIR/"
+cp deployment/index_fetcher.sh "$TEMP_DIR/"
+cp deployment/entry_point.sh "$TEMP_DIR/"
+cp download_latest_release_assets.py "$TEMP_DIR/"
+
+cd "$TEMP_DIR"
+
+# First download and extract remote index server.
+./download_latest_release_assets.py \
+  --repository="kirillbobyrev/indexing-tools" \
+  --asset-prefix="$SERVER_ASSET_PREFIX" \
+  --output-name="$SERVER_ASSET_PREFIX.zip"
```

Comment:
probably cleaner to use `$OUTPUT_NAME` that isn't used anywhere now but that's up to you